### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by holzensp**

### CODE REVIEW

File: .circleci/config.pkl [hunk 1]
```diff
@@ -1,28 +1,106 @@
...
   }
 }
 
-workflows {
-  ["main"] {
-    `when` {
-      equal {
-        #"main"#
-        module.pipelineValues.`pipeline.git.branch`
+// TODO: remove me once policy allows these jobs without approval
```
Comment (bioball) [inline .circleci/config.pkl:81]: I think we can get rid of this now.

File: docs/rules_pkl_docs.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+| <a id="pkl_test_suite-name"></a>name |  A unique name for this target.   |  none |
+| <a id="pkl_test_suite-srcs"></a>srcs |  The source files, containing Pkl test files. Accepts files with the .pkl extension.   |  none |
+| <a id="pkl_test_suite-deps"></a>deps |  Other targets to include in the Pkl module path when building this configuration. Must be pkl_* targets.   |  `None` |
+| <a id="pkl_test_suite-tags"></a>tags |  Tags to add to each Pkl test target.   |  `[]` |
+| <a id="pkl_test_suite-visibility"></a>visibility |  The visibility of non test Pkl source files.   |  `None` |
+| <a id="pkl_test_suite-size"></a>size |  Size of Pkl test.   |  `None` |
+| <a id="pkl_test_suite-test_suffix"></a>test_suffix |  A custom suffix indicating a source file is a Pkl test file.   |  `None` |
+| <a id="pkl_test_suite-kwargs"></a>kwargs |  Further keyword arguments.   |  none |
+
+
```
Comment (bioball) [inline docs/rules_pkl_docs.md:171]: Any plans on making this an Antora module that we can add to pkl-lang.org?

Comment (gibfahn) [reply docs/rules_pkl_docs.md:171]: is this something we can do as a follow-up PR?

**Comment by holzensp** [comment]
... or rather, that my repository was no longer a fork, because it pre-dated turning this repository public.

**Pull request closed without merging**